engo_glfw.go

```diff
@@ -317,7 +323,8 @@ func SetVSync(enabled bool) {
 	}
 }
 
-func init() {
+// Initializes application
+func InitGl() {
 	runtime.LockOSThread()
 
 	Grave = Key(glfw.KeyGraveAccent)
```

Comment:
These should be constant, and not within this method. This is something related to the `input_data.go` file - not PR related, but should change.